src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedVideoConnection.java

```diff
@@ -153,7 +162,25 @@ public void finalizeUpload(FinalizeUploadRequest finalizeUploadRequest) {
         Parameter.with(ACTION_KEY, FINALIZE_UPLOAD));
   }
   
-  private static byte[] convertToBytes(File file) throws IOException {
+  private byte[] convertURLToBytes(URL videoURL) throws IOException {
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    try (InputStream inputStream = videoURL.openStream()) {
+
+      int byteChunk;
```

Comment:
We can specify the bytes size we want to read at a time here - e.g.

byte[] byteChunk = new byte[4096]; 
  int n;

  while ( (n = is.read(byteChunk)) > 0 ) {
    baos.write(byteChunk, 0, n);
  }

Reply:
Good to know, because I know LinkedIn sets limits on the chunks that they accept. It can be part of an improvement to the SDK. Probably more effort than what is required right now, so happy to accept the PR as is.